docs/source/contributing.md

```diff
@@ -0,0 +1,154 @@
+
+In order to contribute via GitHub, you'll need to set up a free account and sign in.
+Here are some [instructions](https://help.github.com/articles/signing-up-for-a-new-github-account/) to help you get going.
+Remember that you can ask us any questions you need to along the way.
+
+## Contribution Model
+
+We have designed our contribution model to be as accessible as possible, while utilising the full power of GitHub's collaboration and version-control tools.
+
+This specification, including its governance procedures and contribution models, are open for the community to evaluate, challenge, amend and discuss.
+If you have improvements we can make to anything we are doing, please suggest them!
+
+The community can suggest any amendments to the specification at any point.
+If you see a part of the specification you don't like, open an issue about it and start a conversation with the community ✨.
+
+:::{important}
+We are in a bootstrapping phase to get an initial specification written.
+As part of this initial work we will propose a more formal governance model for the specification going forwards.
+
+We have chosen to keep all discussion to issues for now, so contributors have a single place to engage in conversation.
+Pull requests are used when a specific change is ready to be proposed.
+This can be without discussion, however, it is best for substantial or significant changes to be discussed first in an issue.
+We have opted to not use GitHub Discussions at this point.
+
+The community can suggest governance changes at any point.
+This includes the SATRE team, and any decision must be openly documented in the repo.
+:::
+
+### Specification Format
+
+The specification source is kept in the [specification repository](https://github.com/sa-tre/satre-specification).
+It is written as a [Sphinx](https://www.sphinx-doc.org/) document in [Markdown](https://www.markdownguide.org/) format.
+
+The most up-to-date 'source of truth' will be the specification on the `main` branch of the specification repository.
+The community can decide when to 'tag' a new version of the specification.
+They may also decide to where to publish the specification.
+
+The specification repository is self-contained and relates only to the specification specifically, or its governance.
+Any contributions to the wider SATRE project should be made through a different medium via the [SATRE GitHub organisation](https://github.com/sa-tre), or by contacting the SATRE team at [satre-contact@dundee.ac.uk](mailto:satre-contact@dundee.ac.uk).
+
+### Contribution Process
+
+Issues should be used to discuss ideas, potential changes and to ask questions.
+Issue templates have been designed for common issue archetypes to help collect the most important information and present it in a clear, consistent way.
```

Comment:
```suggestion
Issue templates have been designed for common issue types to help collect the most important information and present it in a clear, consistent way.
```
I think simple English is best unless "archetypes" conveys something special?

Reply:
Absolutely, as I was writing this I was thinking "Surely I can think of a better word than archetypes" :smile: